spec/requests/api/v1/users/sign_in_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'rails_helper'
+
+RSpec.describe 'User login', type: :request do
+  before do
+    post api_v1_user_session_path, params: user_params, as: :json
+  end
+
+  context 'when user credentials are correct' do
+    let(:password) { Faker::Internet.password }
+    let(:user) { User.create!(attributes_for(:user, password: password)) }
+    let(:user_params) { { email: user.email, password: password } }
```

Comment:
NIT, feel free to not to this one:
We can define `email` as a variable as well, so instead of having to redefine the entire `user_params` whenever we want to change the email we send as param, we could just change that var

```ruby
let(:email) { user.email }
let(:user_params) { { email: email, password: password } }
...
...
...
let(:email) { 'non-existent@mail.com' }
```